components/connection_list/stylesheets/index.styl

```diff
@@ -0,0 +1,56 @@
+borderColor(color)
+  alpha(color, 30%)
```

Comment:
This opacity setting is specific to this connection list component at the moment.